cluster/network.go

```diff
@@ -14,6 +14,10 @@ type Network struct {
 	Engine *Engine
 }
 
+func (network *Network) isPreDefined() bool {
+	return (network.Name == "none" || network.Name == "host" || network.Name == "bridge")
```

Comment:
@dongluochen no, you can create a network with the bridge driver, as you created it, it's not predefined